lib/ezdb/classes/ezdbinterface.php

```diff
@@ -1809,6 +1809,20 @@ public function countStringSize( $string )
      * @var int One of the eZDB::ERROR_HANDLING_* constants
      */
     protected $errorHandling = eZDB::ERROR_HANDLING_STANDARD;
+
+    /**
+     * Maximal value for int columns
+     *
+     * @var int
+     */
+    public $MAX_INT = 2147483647;
```

Comment:
Seems like most reasonable limit supported by every DBMS out-of-the-box.

However these should be constants IMHO // cc @andrerom